.editorconfig

```diff
@@ -0,0 +1,14 @@
+# editorconfig.org
```

Comment:
I think you committed this accidentally

Reply:
For this configuration yes. But why we all can't have the same formatting? The content might be under discussion.

Reply:
Removed.